services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/model/UpdateItemEnhancedRequest.java

```diff
@@ -89,19 +90,17 @@ public boolean equals(Object o) {
         if (o == null || getClass() != o.getClass()) {
             return false;
         }
-
         UpdateItemEnhancedRequest<?> that = (UpdateItemEnhancedRequest<?>) o;
-
-        if (item != null ? ! item.equals(that.item) : that.item != null) {
-            return false;
-        }
-        return ignoreNulls != null ? ignoreNulls.equals(that.ignoreNulls) : that.ignoreNulls == null;
+        return Objects.equals(item, that.item)
+               && Objects.equals(ignoreNulls, that.ignoreNulls)
```

Comment:
I rember null pointer exception if we use  Objects.equals for Boolean , will we face that same issue here ?